cleanlab/multilabel_classification/rank.py

```diff
@@ -110,14 +139,96 @@ def get_label_quality_scores(
     >>> scores
     array([0.9, 0.5])
     """
-
-    assert_valid_inputs(
-        X=None, y=labels, pred_probs=pred_probs, multi_label=True, allow_one_class=True
+    binary_labels = _labels_to_binary(labels, pred_probs)
+    scorer, base_scorer_kwargs = _create_multilabel_scorer(
+        method=method,
+        adjust_pred_probs=adjust_pred_probs,
+        aggregator_kwargs=aggregator_kwargs,
     )
-    num_classes = get_num_classes(labels=labels, pred_probs=pred_probs, multi_label=True)
-    binary_labels = int2onehot(labels, K=num_classes)
-    base_scorer = ClassLabelScorer.from_str(method)
-    base_scorer_kwargs = {"adjust_pred_probs": adjust_pred_probs}
-    aggregator = Aggregator(**aggregator_kwargs)
-    scorer = MultilabelScorer(base_scorer, aggregator)
     return scorer(binary_labels, pred_probs, base_scorer_kwargs=base_scorer_kwargs)
+
+
+def get_class_label_quality_scores(
+    labels: List[List[int]],
+    pred_probs: npt.NDArray["np.floating[T]"],
+    *,
+    method: str = "self_confidence",
+    adjust_pred_probs: bool = False,
+) -> np.ndarray:
```

Comment:
try to make this docstring more clear and what precisely is different from `get_label_quality_scores` method